src/components/search-body/cards-container/MultiSelectColorDropdown.vue

```diff
@@ -0,0 +1,167 @@
+<script lang="ts">
+import HighlightField from '@/components/commons/highlight/HighlightField.vue'
+import { ref, computed, onMounted, watch } from 'vue'
+import { useI18n } from 'vue-i18n'
+import { useParamsStore } from '@/stores/params-store'
+import { useDebounce } from '@/custom-hooks/use-debounce'
+import { ENV } from '@/consts/env.const'
+import { MAX_LENGTH_OF_SEARCH_BY_COLOR } from '@/consts/search-by-color.const'
+import { type ColorProps } from '@/types/color.type'
+
+export default {
+    name: 'MultiSelectColorDropdown',
```

Comment:
```suggestion
    name: 'MultiSelectDropdown',
```
Chỗ này e đã tạo được multiSelectDropdown cho color rồi, giờ e viết cho components được mở rộng hơn  để tái xử dụng được nhé,
Tên sẽ là MultiSelectDropdown
nhận vào các props là listOptions, optionSearch, và setOptions, .. truyền thêm props nếu cần thiết

Những xử lý ở dưới sẽ thực hiện dựa trên những props trên